Source/WebInspectorUI/UserInterface/Views/ObjectTreeArrayIndexTreeElement.css

```diff
@@ -25,7 +25,6 @@
 
 .tree-outline .item.object-tree-array-index {
     position: relative;
-    left: -12px;
```

Comment:
i think this might actually cause the index to be cut off when it's large enough (similar to my concern below about using `min-width`)

Reply:
I reintroduced `left: -12px` (as `inset-inline-start: -12px` to be RTL-compatible). The reason I used `margin-left` instead was to fix the extra spacing that it caused on the right. I offset that with `margin-inline-end: -12px`.